tasks/html.js

```diff
@@ -6,42 +6,104 @@
  * Licensed under the MIT license.
  */
 
+var path = require('path');
+var chalk = require('chalk');
 var htmllint = require('../lib/htmllint');
 
 module.exports = function(grunt) {
   "use strict";
 
-  var chalk = require('chalk');
+  var usingDefaultReporter,
+
+    // Default Grunt reporter
+    defaultReporter = function(result) {
```

Comment:
Let's extract this method and the `selectReporter` method below into a `lib/reporters` module (or similar). Could then also write unit tests for those.